percona-server-8.0/ps-entry.sh

```diff
@@ -227,4 +227,23 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
   fi
 fi
 
+
+CALL_HOME_OPTIONAL_PARAMS=
+
+if [ ! -z "$PERCONA_INSTANCE_ID" ]; then
+  CALL_HOME_OPTIONAL_PARAMS+=" -i ${PERCONA_INSTANCE_ID}"
+fi
+
+if [ ! -z "$PERCONA_TELEMETRY_CONFIG_FILE_PATH" ]; then
+  CALL_HOME_OPTIONAL_PARAMS+=" -j ${PERCONA_TELEMETRY_CONFIG_FILE_PATH}"
+fi
+
+if [ ! -z "$PERCONA_SEND_TIMEOUT" ]; then
+  CALL_HOME_OPTIONAL_PARAMS+=" -j ${PERCONA_SEND_TIMEOUT}"
+fi
+
+# PERCONA_TELEMETRY_DISABLE is handled at the very beginning of call-home.sh
+/call-home.sh -f "PRODUCT_FAMILY_PS" -v "$PS_VERSION" -s "$OS_VER" -d "DOCKER" ${CALL_HOME_OPTIONAL_PARAMS}
```

Comment:
At the moment if script exits with 1 (there is error etc) - the docker does not start + the help is output if smth is wrong with params. I have locally tested with the following command and it works fine:
`/call-home.sh -f "PRODUCT_FAMILY_PS" -v "$PS_VERSION" -s "$OS_VER" -d "DOCKER" ${CALL_HOME_OPTIONAL_PARAMS} &> /dev/null || :`
I would ask @EvgeniyPatlan to check it and comment whether it is ok to be used in Dockerfile.